dotnet/src/Connectors/Connectors.MistralAI/MistralAIToolCallBehavior.cs

```diff
@@ -62,27 +62,24 @@ public abstract class MistralAIToolCallBehavior
     /// <returns>
     /// The <see cref="MistralAIToolCallBehavior"/> that may be set into <see cref="MistralAIPromptExecutionSettings.ToolCallBehavior"/>
     /// to indicate that the specified functions should be made available to the model.
+    /// The model is forced to call a function from the list of functions provided.
     /// </returns>
-    public static MistralAIToolCallBehavior EnableFunctions(IEnumerable<KernelFunction> functions, bool autoInvoke = false)
+    public static MistralAIToolCallBehavior RequiredFunctions(IEnumerable<KernelFunction> functions, bool autoInvoke = false)
     {
         Verify.NotNull(functions);
-        return new EnabledFunctions(functions, autoInvoke);
+        return new AnyFunction(functions, autoInvoke);
```

Comment:
When using Required are you using Auto? Shoudn't it be `RequiredFunction` ?

Reply:
`RequiredFunction` is the name we are using for the behaviour when the model must call a function. MistralAU uses the tool choice "any" for this hence the name `AnyFunction`. But `AnyFunction` is internal and no exposed as part of the public API. 
That said this code will get replaced with Sergeys function calling abstraction code once that is merged.